README.md

```diff
@@ -30,7 +30,7 @@ svn add * --force
 
 7. run 'svn diff' to verify the code changes before you upload the code
 6. Push the changes:
-svn ci -m 'Version `<<new version number>>`' --username Yotpo --password yotpo2010
+svn ci -m 'Version `<<new version number>>`' --username Yotpo --password Y0!po2024

```

Comment:
This is a public repo. Is this okay?

Reply:
Great catch! I don't it is okay to keep SVN pass publicly available, but it was like this so far. I will remove the password here and add confluence page link instead. The question is whether this repo should stay public?

Reply:
I think it is fine to leave this repo public. The plug-in is going to be installed on woocommerce shop and they can look at the code. Also I think this repo needs to be public so the WooCommerce repository can get the new versions.

Reply:
Code of the plugin itself is publicly available here
https://plugins.svn.wordpress.org/yotpo-social-reviews-for-woocommerce/ 
and for sometime here: 
https://github.com/wp-plugins/yotpo-social-reviews-for-woocommerce

As far as I understand, this repo is used only by Yotpo to keep track on changes what is on plugins.svn.wordpress. 

@vladimir-marinov-yotpo Could you please approve the PR?

Reply:
Sure.